financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/Theme.kt

```diff
@@ -158,7 +244,9 @@ private object FinancialConnectionsRippleTheme : RippleTheme {
 internal fun FinancialConnectionsTheme(content: @Composable () -> Unit) {
     CompositionLocalProvider(
         LocalFinancialConnectionsTypography provides Typography,
-        LocalFinancialConnectionsColors provides LightColorPalette
+        LocalV3Typography provides V3Typography,
+        LocalFinancialConnectionsColors provides Colors,
+        LocalV3Colors provides V3Colors
     ) {
```

Comment:
I assume that the V3 colors and typography are going to be the only ones at some point, right? Would it make sense to have a local feature flag here to decide between both?

```suggestion
internal fun FinancialConnectionsTheme(content: @Composable () -> Unit) {
  val colors = if (FeatureFlag.v3Theme.isEnabled) V3Colors else Colors
  
  CompositionLocalProvider(
    LocalFinancialConnectionsTypography provides Typography,
    LocalFinancialConnectionsColors provides LightColorPalette,
  ) {
}
```

Thinking further, you could also easily have this be backend-driven and allow switching themes in the sample app.

Reply:
They'll all go away before merging the integration branch. The approach will be:

- Introduce all new theme related options
- Redesign each pane in the flow (on separate PRs) 
- Delete old theme related code.

> you could also easily have this be backend-driven and allow switching themes in the sample app.

We currently don't support dark theme / theme customizations, but introducing them on the [Manifest](https://github.com/stripe/stripe-android/blob/1bdd5d6f7343adbc857d237db4bb52b0b8046dd1/financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsSessionManifest.kt#L53) fetched at the start of the flow is a great idea - I'll propose it!

Reply:
we will be transitioning to v3 directly, merging the integration branch once all the design changes (colors + fonts + pane redesign) is all ready. Some more info about it:

- [Kick off doc](https://docs.google.com/document/d/1g7sUNkb2wRLUJSir6vY4x4dLvfeYGn9U1__Wc_-jtmk/edit#heading=h.b256c98r8msg)
- [Figma](https://www.figma.com/file/wXQ8NJApqSJiLmxxANDRTs/%E2%9C%A8-Connections-3.0?type=design&node-id=0%3A1&mode=dev)